tests/serverpod_test_server/lib/src/endpoints/async_tasks.dart

```diff
@@ -1,3 +1,5 @@
+// ignore_for_file: deprecated_member_use
```

Comment:
I think here we either test deprecated stuff, or we should test no deprecated stuff. New tests should be written against the new interface and probably live in another file.

Reply:
But since we have mixed the code paths, would this file in its entirety be safe to delete without losing coverage?

If not, we should aim to have the `deprecated_member_use` ignore removed at some point.

Reply:
updated to new methods.